docs/webpack.md

```diff
@@ -0,0 +1,107 @@
+# Webpack usage examples
+
+## Install yargs
+
+Install yargs as a production dependency, as it will _not_ be included in the pack (not yet compatible):
```

Comment:
Yet some people could _want_ to ignore webpack warnings and have it included in the pack.

I am adding this.

Reply:
Fixed